README.md

```diff
@@ -15,14 +15,36 @@ SimpleX chat prototype is a thin terminal UI on top of [SimpleXMQ](https://githu
 
 See [simplex.chat](https://simplex.chat) website for chat demo and the explanations of the system and how SMP protocol works.
 
+### Quick installation
+
+```sh
+curl -o- https://raw.githubusercontent.com/simplex-chat/simplex-chat/master/install.sh | bash
+```
+
 ![simplex-chat](./images/connection.gif)
 
+### Welcome word
+
+**We are working hard to create the most secure and user privacy focused chat in the world** - open source, decentralized, and without the need for global identities of any kind. If you believe in the project and would like to support it, you can do so in following ways:
+
+- 🌟 **Give the project a star on GitHub** - it's a great feedback we're on the right track and helps us raise the visibility of the project.
+
+- **Install the chat and try it out** - if you find a bug you may help us out by [raising an issue](https://github.com/simplex-chat/simplex-chat/issues).
```

Comment:
```suggestion
- **Install the chat and try using it** - if you spot a bug it would help a lot if you could [raise an issue](https://github.com/simplex-chat/simplex-chat/issues).
```